api/app/Services/YandexDiskService.php

```diff
@@ -58,37 +58,37 @@ public function createFolder(
             } else {
                 $folderPath = '/'.$parentId.'/'.$name;
             }
-
+            
             $result = $this->disk->getResource($folderPath)->create();
-
+            
             var_dump($result->toArray());
             if ($result) {
                 return $folderPath;
             } else {
                 throw new Exception('Can\'t Create disk folder');
             }
         }
-
+        
         /**
          * @inheritDoc
          */
         public function getChildFolderByName(string $folderID, string $subFolderName, bool $toRetry = true): ?string
         {
             return '/'.$subFolderName.'/';
         }
-
+        
         /**
          * @inheritDoc
          * @throws Exception
          */
         public function listFolderImages(string $folderID, bool $toRetry = true): array
         {
             $folderPath = $this->cleanupPath("/".$folderID);
-
+            
             try
             {
-                $directory = $this->disk->getResource($folderPath);
-
+                $directory = $this->disk->getResource($folderPath, 1000);
```

Comment:
Is there a reason to stop at exactly 1000 files?

Reply:
Customer preference